integration_tests/integration/excludedRestricted.spec.ts

```diff
@@ -11,15 +9,6 @@ context('Excluded and restricted cases', () => {
   })
 
   context('Excluded', () => {
-    it('search results page', () => {
```

Comment:
Do we not need some sort of equivalent of this test for search by name?

Reply:
no - since the endpoint now returns an array - it can't complain about excluded cases anymore.  We encounter that error when we arrive on the case overview.